devops/resources/table-size-monitoring.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+cd $WORKSPACE/configuration/util/jenkins/check_table_size
+
+pip install -r requirements.txt
+. ../assume-role.sh
+
+# Assume the role
+set +x
+assume-role ${ROLE_ARN}
+set -x
+
+# Set RDSTHRESHOLD if not set in job, need because we're setting -u
+# Otherwise we get an error "RDSTHRESHOLD: unbound variable"
+if [[ ! -v RDSTHRESHOLD ]]; then
+    RDSTHRESHOLD=""
+fi
+
+python check_table_size.py  --username ${USERNAME} --password ${PASSWORD} --threshold ${THRESHOLD} ${RDSTHRESHOLD}
```

Comment:
I would load the password from an environment variable instead of expecting it as a passed arg, as this makes it known to anyone on the system who can inspect the running processes.

Reply:
aka other jenkins tasks